LoopKitUI/Views/Information Screens/GlucoseTherapySettingInformationView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  GlucoseTherapySettingInformationView.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 11/16/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+import SwiftUI
+import LoopKit
+
+public struct GlucoseTherapySettingInformationView: View {
+    var text: AnyView!
+    let onExit: (() -> Void)?
+    let mode: PresentationMode
```

Comment:
[nit] Some confusion between `PresentationMode` type here and `presentationMode` in environment. I understand the intention after reading the code, but it gave me pause at first.

Reply:
Ick.  Yeah, that name collision has always been there.  I think I'll rename the former "SettingsPresentationMode"